integration/fuse/src/main/java/alluxio/fuse/AlluxioJniFuseFileSystem.java

```diff
@@ -679,8 +690,8 @@ private int chownInternal(String path, long uid, long gid) {
 
   @Override
   public int truncate(String path, long size) {
```

Comment:
Why we change truncate to 0 return?
what's the return int means in this function? it means the operation succeeds or not or how many bytes we truncate?

Reply:
The return of `truncate` is error code, 0 stand for success. It would be better to avoid non-zero size truncate.